vector/v.info/parse.c

```diff
@@ -56,4 +59,9 @@ void parse_args(int argc, char **argv, char **input, char **field, int *history,
         *shell |= SHELL_REGION;
     if (topo_flag->answer)
         *shell |= SHELL_TOPO;
+
+    if (strcmp(format_opt->answer, "json") == 0)
```

Comment:
@wenzeslaus About renaming from PLAIN to SHELL for future clarity, please note that we assign format to SHELL/PLAIN if its not set to json and the standard option takes only PLAIN and JSON as input.

Reply:
Good point, we actually have a deeper problem here. I think it was different in my head and yours and I didn't catch that in the PRs. Considering v.db.select, discussion I had with @petrasovaa, and interface or r.info, we need to have:

* format=plain: human readable format (optimized for humans, should not be parsed)
* format=shell: what we call "shell script style" (easy to manually parse in Bash, can be passed to eval in Bash)
* format=json: JSON

So, in code it is PLAIN, JSON, SHELL, i.e., one more format comparing to what you have there.

Reply:
@wenzeslaus I was planning to update the standard format option to accommodate a shell option too but I realized that it not all modules have different plain and shell style outputs. The ones which do have both often have a separate flag to toggle between the outputs. 

Should I update the standard format option and try to update its existing usage or should I just update the option for this module for now?

Reply:
Just update it for this module to move forward. We can go back and adjust the standard option if needed - by that time we will also have an idea what is more common and thus what should be the default. Sorry for the late answer.

Reply:
Makes sense, thanks I have updated the module.